components/tyk-mdcb/templates/deployment-mdcb.yaml

```diff
@@ -115,6 +115,16 @@ spec:
             - name: TYK_MDCB_ANALYTICSCONFIG_MONGOUSESSL
               value: "{{ default "false" .Values.global.mongo.useSSL }}"
             {{ end }}
+            {{ if .Values.global.mdcbSynchronizer.enabled }}
+            - name: TYK_MDCB_SYNCWORKER_ENABLED
+              value: {{ .Values.global.mdcbSynchronizer.enabled | quote }}
+            - name: TYK_MDCB_SYNCWORKER_HASHKEYS
+              value: {{ .Values.mdcb.hashKeys | quote }}
+            {{ if .Values.global.mdcbSynchronizer.privateCertificateEncodingSecret }}
```

Comment:
This field is not specific to Synchronizer. It need to be set if certificates are used.

Reply:
waiting for TLS ticket https://github.com/TykTechnologies/tyk-charts/pull/219 to be merged. then, I'll update according to your suggestion